admin/partials/nginx-helper-general-options.php

```diff
@@ -311,9 +322,12 @@
 										echo wp_kses(
 											sprintf(
 												'%1$s<strong>%2$s</strong>%3$s<strong>%4$s</strong>.',
-												esc_html__( 'when a ', 'nginx-helper' ), esc_html__( 'published post', 'nginx-helper' ), esc_html__( ' (or page/custom post) is ', 'nginx-helper' ), esc_html__( 'trashed', 'nginx-helper' )
+												esc_html__( 'when a ', 'nginx-helper' ),
+												esc_html__( 'published post', 'nginx-helper' ),
+												esc_html__( ' (or page/custom post) is ', 'nginx-helper' ),
+												esc_html__( 'trashed', 'nginx-helper' )
```

Comment:
This is not correct way to make small string translatable. We need to allow entire string translatable.